packages/jsx-directive/src/core/index.ts

```diff
@@ -15,7 +16,10 @@ export function transformJsxVueDirective(code: string, id: string) {
     ast: Program
     offset: number
   }[] = []
-  if (lang === 'vue') {
+  if (
+    lang === 'vue' ||
+    (REGEX_SETUP_SFC.test(id) && code.startsWith('<script '))
```

Comment:
Why check `&& code.startsWith('<script ')` here?

Reply:
for test `v-else-if.setup.tsx`

Reply:
how about change to `v-else-if.tsx` for tests